usecases/transfer_check_usecase.go

```diff
@@ -87,16 +113,29 @@ func (usecase *TransferCheckUsecase) CreateTransfer(
 		transferMappingId = uuid.New().String()
 		err = usecase.transferMappingsRepository.CreateTransferMapping(ctx, exec,
 			transferMappingId, models.TransferMappingCreateInput{
-				OrganizationId:   organizationId,
 				ClientTransferId: transfer.TransferData.TransferId,
+				OrganizationId:   organizationId,
+				PartnerId:        partnerId,
 			})
 		if err != nil {
 			return models.Transfer{}, err
 		}
+		transferMappings, err = usecase.transferMappingsRepository.ListTransferMappings(
+			ctx,
+			exec,
+			organizationId,
+			partnerId,
+			transfer.TransferData.TransferId,
+		)
```

Comment:
In this perticular case, you can directly use `GetTransferMapping` since you generate `transferMappingId = uuid.New().String()`

Reply:
ah yes, thank you